idemp-base/src/org/bandahealth/idempiere/base/model/MBandaSetup.java

```diff
@@ -1359,6 +1360,74 @@ public boolean configureClientUsers() {
 		});
 		return true;
 	}
+	
+	/**
+	 * Create default business partners for new clients
+	 * @return
+	 */
+	public boolean createDefaultBusinessPartners() {
+		List<MBPartner_BH> businessPartners = new Query(this.context, MBPartner_BH.Table_Name,
+				MBPartner_BH.COLUMNNAME_AD_Client_ID + "=?", getTransactionName()).setParameters(MClient_BH.CLIENTID_CONFIG)
+				.list();
+		businessPartners.forEach((businessPartner) -> {
+			MBPartner_BH instance = new MBPartner_BH(context, 0, getTransactionName());
```

Comment:
Can you just use do:
```
MBPartner_BH instance = MBPartner_BH.copyValues(businessPartner, new BPartner_BH(context, 0, getTransactionName()));
```

Reply:
Getting cast exceptions because `copyValues` defined in `PO` returns `MBPartner`. Should we define our own `copyValues` in `MBPartner_BH` or should we cast the returned object to `MBPartner_BH`? Thanks

Reply:
Lame. If you just want to use `MBPartner`, that's okay (all the values will still get copied correctly). We'd only need the `MBPartner_BH` class if we're trying to use our specific get methods.

Reply:
Done